### ISSUE TITLE
Cannot redefine property: inputs

### ISSUE THREAD

**Issue by PetrosiliusPatter** [emilwidlund/nodl#12]
I wrote a custom Node, just like in the demo sandbox. However, when trying to create it with `new Addition()`, I get this error: `Uncaught TypeError: Cannot redefine property: inputs`.

However, to make things weirder: I'm using a local fork of nodl, which I linked with yarn into my project. When defining the node inside the nodl-fork, and import it in my own project, it works! And running the playground in the forked nodl works as well, obviously. It only breaks once I define the node in own project. 

Have you come across something like this?

**Comment by emilwidlund** [comment]
@PetrosiliusPatter Are you able to reproduce this in a CodeSandbox or similar that you can link me?

**Comment by jmpinit** [comment]
I had the same issue after setting up a React project with Webpack and Babel. I pasted [the demo code](https://codesandbox.io/s/agitated-elgamal-txqqql?file=/src/App.js) into my app and then had the exact same error on my page.

I found a solution in using the [transform-class-properties](https://www.npmjs.com/package/babel-plugin-transform-class-properties) Babel plugin. After adding that to my Babel config the demo code works in my project.
